tests/app/shared/issue-tables/issues-table-paginated-data.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { MatPaginator } from "@angular/material";
+import { Issue } from "../../../../src/app/core/models/issue.model";
+import { TEAM_4 } from "../../../constants/data.constants";
+import {
+    ISSUE_WITH_ASSIGNEES,
+    ISSUE_WITH_EMPTY_DESCRIPTION,
+    ISSUE_WITH_EMPTY_DESCRIPTION_HIGH_SEVERITY,
+    ISSUE_WITH_EMPTY_DESCRIPTION_LOW_SEVERITY
+} from "../../../constants/githubissue.constants";
+import { paginateData } from '../../../../src/app/shared/issue-tables/issue-paginator';
+
+describe('isses-table-paginated-data', () => {
```

Comment:
Do rename 
```suggestion
describe('issue-paginator', () => {
```

Do rename the test file, so it's in line with the name of the module

Reply:
Sorry, i must have missed out on it😅